src/widgets/customapi/component.jsx

```diff
@@ -245,7 +245,11 @@ export default function Component({ service }) {
               listItems.map((item, index) => {
                 const itemName = shvl.get(item, name, "");
                 const itemLabel = shvl.get(item, label, "");
-                const itemUrl = target ? target.replace(/\{([^}]+)\}/g, (_, key) => item[key] || "") : null;
+
+                const itemUrl = target ? [...target.matchAll(/\{(.*?)\}/g)].map((match) => match[1]).targetReplaces.reduce((url, targetTemplate) => {
+                  const value = shvl.get(item, targetTemplate, "");
```

Comment:
```suggestion
                  const value = shvl.get(item, targetTemplate, item[targetTemplate]);
```

would fallback to getting the prop if it has a '.'